src/gsCore/gsLinearAlgebra.h

```diff
@@ -18,7 +18,7 @@
 
 // Eigen linear algebra library (http://eigen.tuxfamily.org)
 
-#define Eigen gsEigen
```

Comment:
I would propose to add here a namespace alias:

namespace gsEigen = Eigen;	

or something like: #define gsEigen Eigen

and always use inside our sources gsEigen instead of Eigen.

This will allow the possibility to rename the symbols globally if some developer has problems with Eigen (see issues Issue #601 and #438).